CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.4.0 (2019-01-08)
+
+- Added invalid field status styles to textarea input. [#46](https://github.com/blackbaud/skyux-theme/pull/46)
+- Added support for `@blackbaud/skyux-design-tokens@0.0.11`. [#45](https://github.com/blackbaud/skyux-theme/pull/45)
```

Comment:
Is the second "@" the correct way to indicate the particular version?

Reply:
Yep! At least from the perspective of the NPM command line.
```
npm install @blackbaud/skyux-design-tokens@0.0.11
```

Reply:
Great. Thanks.